src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -167,82 +167,58 @@ ServerHandlerImp::onAccept (Session& session,
     return true;
 }
 
-auto
-ServerHandlerImp::onHandoff (Session& session,
+Handoff
+ServerHandlerImp::onHandoff(
+    Session& session,
     std::unique_ptr <beast::asio::ssl_bundle>&& bundle,
-        http_request_type&& request,
-            boost::asio::ip::tcp::endpoint remote_address) ->
-    Handoff
+    http_request_type&& request,
+    boost::asio::ip::tcp::endpoint const& remote_address)
```

Comment:
Is `endpoint` a small enough type to keep it as a value type or worth taking a reference to?

Reply:
It might be, but I think a const ref is well suited here.